web/components/new-contract/cost-section.tsx

```diff
@@ -82,45 +99,34 @@ function PriceSection(props: {
         Choose a tier to determine how much initial liquidity to inject into the
         market. More liquidity attracts more traders but has a higher cost.
       </div>
-      <div
-        className={clsx(
-          'grid w-full gap-2',
-          outcomeType === 'NUMBER'
-            ? 'grid-cols-3'
-            : 'grid-cols-2 sm:grid-cols-5'
-        )}
-      >
-        {outcomeType !== 'NUMBER' && (
-          <Tier
-            baseCost={baseCost}
-            tier="play"
-            icon={<PlayTier />}
-            outcomeType={outcomeType}
-            currentTier={currentTier}
-            setMarketTier={setMarketTier}
-          />
-        )}
-        {outcomeType !== 'NUMBER' && (
-          <Tier
-            baseCost={baseCost}
-            tier="basic"
-            icon={
-              <LogoIcon
-                className="stroke-ink-600 flex-inline shrink-0 stroke-[1.5px]"
-                aria-hidden
-                style={{
-                  width: '1em',
-                  height: '1em',
-                  marginRight: '0.1em',
-                  marginBottom: '0.1em',
-                }}
-              />
-            }
-            outcomeType={outcomeType}
-            currentTier={currentTier}
-            setMarketTier={setMarketTier}
-          />
-        )}
+      <div className="grid w-full grid-cols-2 gap-2 sm:grid-cols-5">
+        <Tier
+          baseCost={baseCost}
+          tier="play"
+          icon={<PlayTier />}
+          outcomeType={outcomeType}
+          currentTier={currentTier}
+          setMarketTier={setMarketTier}
```

Comment:
It would be nice to put a isTierDisabled boolean here and then have the disabled tier UX in the Tier component. Not a blocking thing, I can do it after you merge

Reply:
Makes sense, thank you!